src/core/inferFullTargets.ts

```diff
@@ -75,6 +75,7 @@ function inferRangeTarget(
     type: "range",
     excludeAnchor: target.excludeStart ?? false,
     excludeActive: target.excludeEnd ?? false,
+    rangeType: target.rangeType,
```

Comment:
Should we default this one to `"horizontal"` or `"standard"` or `"contiguous"` or something like that?

Reply:
We could do that if you prefer. The problem is that all existing tests need to be updated and I don't know how to do that yet.

Reply:
Agreed we can default it in the inference stage